docs/api/_src/api/pipelines/index.md

```diff
@@ -0,0 +1,8 @@
+# API Reference
```

Comment:
Shouldn't this title be `pipeline lint test`? As it's written [here](https://github.com/nf-core/tools/pull/3054/files#diff-4771afee2d8066fa3be4ba4965a28cd8afd54a623054e758a5f5182fd1ed0405R48)

Reply:
for "historical" reasons we have all pipeline commands there as an API reference https://nf-co.re/docs/nf-core-tools/api_reference/dev/api/bump_version. not sure if we want to keep them or add the modules and subworkflow commands as well...

Reply:
ah ok I was confused between linting tests and commands, let's leave it like it is for now